.changeset/grumpy-deers-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': patch
+---
+
+Delayed transitions defined using `after` were previously causing a circular dependency when the machine was converted `.toJSON()`. This has now been fixed.
```

Comment:
```suggestion
Delayed transitions defined using `after` were previously causing a circular dependency when the machine was converted using `.toJSON()`. This has now been fixed.
```

not sure if this fixes grammar or not, feel free to ignore the suggestion - you are the native English speaker so you know best 😅